README.rst

```diff
@@ -1246,7 +1246,7 @@ See the `Mutual Authentication section <#813-mutual-authentication>`__.
 
 Hazelcast Python client can discover and connect to Hazelcast clusters
 running on `Hazelcast Cloud <https://cloud.hazelcast.com/>`__. For this,
-provide authentication information as ``cluster_name``, enable cloud
+provide authentication information as ``cluster_name`` and enable cloud
```

Comment:
Can't comment directly on the line, so I'm leaving the comment here.

nit: L1262 should use `Hazelcast Python client` instead of `Hazelcast Python Client`.

Reply:
Fixed it